src/copyist/helpers.py

```diff
@@ -0,0 +1,20 @@
+import tomlkit
+
+
+def ensure_tool_table(previous_content, tool_name, table_text):
```

Comment:
```suggestion
def fill_tool_section(previous_content, tool_name, section_text):
```

I am not fond of the name `ensure_tool_table` for two reasons:

- "ensure" make it seem like the function returns a boolean, i.e. whether `previous_content` already contains `table_text`;
- `table_text` makes sense if someone knows what a table is in TOML, but not so much otherwise. (Disclosure: I just read the specs for the first time.)

"section" looks more understandable than "table". Also, this function *fills* content. Hence `fill_tool_section()`.

Anyway, whichever name we choose, we'll add a docstring later. :)